app.js

```diff
@@ -1,55 +1,313 @@
+  app.get("/", (req, res) => {
+    res.render("index.html", {
+      locals: app.locals
+    });
+  });
+
+  var appPages = {
+    "/clock-color.html": "clock-color.html"
+    , "/admin": "adminindex.html"
+    , "/profile": "profileindex.html"
+    , "/food": "foodindex.html"
+    , "/bgclock.html": "bgclock.html"
+    , "/report": "reportindex.html"
+    , "/translations": "translationsindex.html"
+    , "/clock.html": "clock.html"
+  };
+
+  Object.keys(appPages).forEach(function(page) {
+    app.get(page, (req, res) => {
+      res.render(appPages[page], {
+        locals: app.locals
+      });
+    });
+  });
+
+  app.get("/appcache/*", (req, res) => {
+    res.render("nightscout.appcache", {
+      locals: app.locals
+    });
+  });
+
+  app.use('/api/v1', bodyParser({
+    limit: 1048576 * 50
+  }), api);
 
+  app.use('/api/v2', bodyParser({
+    limit: 1048576 * 50
+  }), api);
+
+  app.use('/api/v2/properties', ctx.properties);
+  app.use('/api/v2/authorization', ctx.authorization.endpoints);
+  app.use('/api/v2/ddata', ddata);
+  app.use('/api/v2/notifications', notificationsV2);
+
+  app.use('/api/v3', api3);
 
   // pebble data
   app.get('/pebble', ctx.pebble);
 
+  // expose swagger.json
+  app.get('/swagger.json', function(req, res) {
+    res.sendFile(__dirname + '/swagger.json');
+  });
+
   // expose swagger.yaml
-  app.get('/swagger.yaml', function (req, res) {
+  app.get('/swagger.yaml', function(req, res) {
     res.sendFile(__dirname + '/swagger.yaml');
   });
 
-  //app.get('/package.json', software);
+  if (env.settings.isEnabled('dumps')) {
+    var heapdump = require('heapdump');
+    app.get('/api/v2/dumps/start', function(req, res) {
+      var path = new Date().toISOString() + '.heapsnapshot';
+      path = path.replace(/:/g, '-');
+      console.info('writing dump to', path);
+      heapdump.writeSnapshot(path);
+      res.send('wrote dump to ' + path);
+    });
+  }
+
+  // app.get('/package.json', software);
+
+  // Allow static resources to be cached for week
+  var maxAge = 7 * 24 * 60 * 60 * 1000;
+
+  if (process.env.NODE_ENV === 'development') {
+    maxAge = 1;
+    console.log('Development environment detected, setting static file cache age to 1 second');
 
-  // define static server
-  //TODO: JC - changed cache to 1 hour from 30d ays to bypass cache hell until we have a real solution
-  var staticFiles = express.static(env.static_files, {maxAge: 60 * 60 * 1000});
+    app.get('/nightscout.appcache', function(req, res) {
+      res.sendStatus(404);
+    });
+  }
+
+  var staticFiles = express.static(env.static_files, {
+    maxAge
+  });
 
   // serve the static content
   app.use(staticFiles);
 
-  var bundle = require('./bundle')();
-  app.use(bundle);
+  // API docs
+
+  const swaggerUi = require('swagger-ui-express');
+  const swaggerDocument = require('./swagger.json');
+
+  app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
+
+  app.use('/swagger-ui-dist', (req, res, next) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['next' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)